exercises/practice/binary-search-tree/.meta/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "authors": [],
```

Comment:
Could you add yourself as the author?

Reply:
I would prefer to leave this blank. Is that alright, or is it necessary?

Reply:
That's fine